html/elements/video.json

```
@@ -264,12 +264,28 @@
               "edge": {
                 "version_added": "≤18"
               },
-              "firefox": {
-                "version_added": "12"
-              },
-              "firefox_android": {
-                "version_added": "14"
-              },
+              "firefox": [
+                {
+                  "version_added": "74"
+                },
+                {
+                  "version_added": "12",
+                  "version_removed": "74",
+                  "partial_implementation": true,
+                  "notes": "See Bugzilla <a href='https://bugzil.la/1532722'>bug 1532722</a>."
```

Two suggestions in one here: first, to briefly summarize how the implementation is incomplete and second, very minor, to not bother naming Bugzilla for consistency with other nodes in BCD

```suggestion
                  "notes": "With <code>crossorigin=\"use-credentials\"</code>, cookies aren't sent during seek. See <a href='https://bugzil.la/1532722'>bug 1532722</a>."
```